src/libcollections/str.rs

```diff
@@ -789,27 +802,37 @@ impl str {
         core_str::StrExt::split_at(self, mid)
     }
 
-    /// An iterator over the codepoints of `self`.
+    /// An iterator over the Unicode scalar values of `self`.
+    ///
+    /// In Unicode a single letter can be composed of multiple `char`s,
+    /// and a single `char` can represent multiple letters.
+    ///
+    /// For iteration over human-readable characters a [grapheme cluster iterator][1]
+    /// may be more appropriate.
+    ///
+    /// [1]: https://unicode-rs.github.io/unicode-segmentation/unicode_segmentation
```

Comment:
This link doesn't actually give the reader any indication where to find & download the crate -- crates.io might be more appropriate. I don't know however what we usually do in this situation, do we link out to third party crates?